docs/payments/paymentMethods.mdx

```diff
@@ -70,7 +70,7 @@ customer preferences and transaction scenarios, all while maintaining security a
 
 ## Payment Method Groups (PMG)
 
-Payment Method Groups offer a way to easily assign available payment methods for a checkout session 
+Payment Method Groups offer a way to easily assign available payment methods to an entity (e.g. a wallet on your platform)
```

Comment:
Does this make sense?

Reply:
Not to me, we don't have that concept yet, but I can also see it as just an example. I think we should remove to avoid confusion